### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tan-ro**
New README file: Added links to specific producs, added to support disclaimer, updated installation links.

### CODE REVIEW

File: docs/CONFIGURATION.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Configuration #
```
Comment (nanux) [inline docs/CONFIGURATION.md:1]: Do we want to use the format with opening and closing header markdown going forward? `# header #` vs `# header`

It is very minor but I am for consistency.

File: README.md [hunk 2]
```diff
@@ -4,27 +4,43 @@
...
 
-This project contains [Helm charts](https://helm.sh/) for Atlassian's [Data Center products](https://www.atlassian.com/enterprise/data-center).
-The charts will allow the Data Center products to be easily installed and operated within a Kubernetes cluster,
-whether it be a managed environment such as Amazon EKS, or a custom, on-premise system.
+This project contains [Helm charts](https://helm.sh/) for installing Atlassian's [Jira Data Center](https://www.atlassian.com/enterprise/data-center/jira), [Confluence Data Center](https://www.atlassian.com/enterprise/data-center/confluence), and [Bitbucket Data Center](https://www.atlassian.com/enterprise/data-center/bitbucket) on [Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/). 
+
+Use the charts to easily install and operate Data Center products within a Kubernetes cluster of your choice. It can be a managed environment, such as Amazon EKS and Azure Kubernetes Service (AKS), or a custom on-premise system.
```
Comment (nanux) [inline README.md:9]: I think we can mention `Amazon EKS, Azure AKS or Google GKE`.

File: docs/INSTALLATION.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+Use these instructions to install your Atlassian product using the Helm charts. Make sure you have followed the [Prerequisites guide](PREREQUISITES.md) before you proceed with the installation.
+
+#### 1. Add the Helm chart repository to your local Helm installation: ####
+
+Add the Helm chart repository to your local Helm installation:
+
+   `helm repo add atlassian-data-center https://atlassian-labs.github.io/data-center-helm-chart`
+
+   * Substitute `atlassian-data-center` with whatever name you wish
```
Comment (nanux) [inline docs/INSTALLATION.md:11]: I think we can drop this if people want to change it - they will probably know how to do it.

File: docs/PREREQUISITES.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Prerequisites #
+## Requirements ##
+
+In order to deploy Atlassian’s Data Center products, the following is required:
```
Comment (nanux) [inline docs/PREREQUISITES.md:4]: I think we should also include the expectation of a certain understanding of Kubernetes and Helm concepts.

File: docs/INSTALLATION.md (hunk 3 above)
Comment (tan-ro) [reply docs/INSTALLATION.md:11]: Agree

**Review by nanux — APPROVED** [review]

**Pull request merged**